composer.json

```diff
@@ -197,6 +197,7 @@
             "App\\Environment::checkEnvironment",
             "@auto-scripts"
         ],
+        "security-check": "security-checker security:check",
```

Comment:
not conencted to this line:

```
"auto-scripts": {
            "php phing clean": "script",
            "shopsys:domains-urls:configure": "symfony-cmd",
            "ckeditor:install --clear=skip --release=full --tag=4.5.11": "symfony-cmd",
            "security-checker security:check": "script"  ### this line bothers me
        }
```

Could we use `"composer security-check": "script"` to prevent DRY?

Reply:
I'm not sure. If the composer is not available globally (for any reason), this auto-script will fail then

Reply:
I do not understand it now. If it might not be reliable why are we using the composer script? What I am suggesting as the same way as it is called from phing.

Reply:
I see. You're right it's used in Phing anyway. fixed in ed8fbe8c2